.github/workflows/preview.yml

```diff
@@ -22,29 +22,30 @@ jobs:
         with:
           version: '1.10'
 
+      - name: Instantiate Julia environment
+        run: julia --project=. -e 'using Pkg; Pkg.instantiate()'
```

Comment:
We were instantiating environment in quarto file itself so we will do it this way then that should be removed I guess! See https://github.com/TuringLang/docs/blob/1f797b78268685c853efb5a32c3c71744152e47b/tutorials/00-introduction/index.qmd#L8C1-L13C4

Reply:
Also, I can't see project instantiation in publish workflow!

Reply:
Also, I think we did tried this approach at first, and we faced some issues back then, I don't remember the particulars but this time it looks fine to me, let's try it!

Reply:
Oh, yes, you're right! Still need to instantiate in the publish workflow.

If it's a single env then instantiating it one time at the start works, I tested it [once here](https://github.com/TuringLang/docs/actions/runs/10131613167/job/28014548724?pr=497) :)